### ISSUE TITLE
Bluetooth Controller isn't present - Pi400

### ISSUE THREAD

**Issue by egandro** [home-assistant/operating-system#1469]
- [X] Raspberry Pi [1/2/3/4] - Pi400

OS Version:               Home Assistant OS 6.1
Home Assistant Core:      2021.7.3

```
~ $ bluetoothctl
Agent registered
[bluetooth]# list
[bluetooth]# power on
No default controller available
```

The default Pi 400 controller isn't available. Maybe there are outdated firmwares on the OS image?

**Comment by agners** [comment]
This was a missing firmware. It will be fixed in 6.2 and later.

**Comment by egandro** [comment]
Cheers @agners you deserve a :beer:
